crates/bevy_ecs/macros/Cargo.toml

```diff
@@ -11,7 +11,7 @@ proc-macro = true
 [dependencies]
 bevy_macro_utils = { path = "../../bevy_macro_utils", version = "0.14.0-dev" }
 
-syn = { version = "2.0", features = ["full"] }
+syn = "2.0"
```

Comment:
This is a bad merge from main where the same thing was fixed. I think the merge might go through if this change is removed.